.github/workflows/api.yaml

```
@@ -11,16 +11,27 @@ jobs:
+        
+      - uses: astral-sh/setup-uv@v6
         with:
-          project-path: ./api
-          enable-cache: "true"
+          enable-cache: true
+          uv-version: ${{ steps.uv-version.outputs.version }}
+
       - name: Install dependencies
-        run: devbox run -- uv sync
+        run: uv sync
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)